certbot.wrapper

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
nit: `/bin/sh` seems to work, but I think the behavior of commands like `echo -n` depends on the system the script is run on. Explicitly using bash should fix this, I tested it in the snap, and it worked.
```suggestion
#!/bin/bash
```